config/settings/base.py

```diff
@@ -412,11 +412,13 @@
     SPRINT_STATUS_IN_DEVELOPMENT,
 }
 # String for overriding how much time will be needed for an epic management per sprint.
-SPRINT_EPIC_DIRECTIVE = fr"\[~{JIRA_BOT_USERNAME}\]: plan (\d+) hours per sprint for epic management"
+SPRINT_EPIC_DIRECTIVE = fr"\[~{JIRA_BOT_USERNAME}\]: plan (\d*)\.?(\d*) hours per sprint for epic management"
```

Comment:
Thank you, @xitij2000! It looks much more user-friendly.
I simplified it a bit to `(?:(?P<hours>\d+)\s?h.*?)?\s?(?:(?P<minutes>\d+)\s?m.*?)?`.

_Nit_: it will also match `…plan  per…` (with two spaces between these words - GitHub renders only one), but I suppose we can ignore this to avoid making this regex less readable. If it's important, we can just add a positive lookahead for verifying it (e.g. `(?=.*\d)`).

Reply:
Sure! My general idea is just about being able to specify in hours and minutes instead of decimal time. In my case I only hatch h and m, since I control the format. However here we'll need to continue matching the existing pattern.